src/Adapter/Language/CommandHandler/AbstractLanguageHandler.php

```diff
@@ -144,4 +147,29 @@ protected function getLegacyLanguageObject(LanguageId $languageId)
 
         return $language;
     }
+
+    /**
+     * @param Language $language
+     */
+    protected function assertLanguageIsNotInUse(Language $language)
+    {
+        if ($language->id === (int) Context::getContext()->language->id) {
+            throw new DefaultLanguageException(sprintf('Used language "%s" cannot be deleted', $language->iso_code), DefaultLanguageException::CANNOT_DELETE_IN_USE_ERROR);
+        }
+    }
+
+    /**
+     * @param Language $language
+     * @param ToggleLanguageStatusCommandInterface $command
+     */
+    private function assertLanguageIsNotDefault(Language $language, ToggleLanguageStatusCommandInterface $command = null)
```

Comment:
If you want to use this in other classes it needs to be `protected` no?

Reply:
Exactly ! But it's very strange how Travis didn't catch it ?